modules/ocf_desktop/manifests/xfce.pp

```diff
@@ -3,7 +3,6 @@
     recommends => false;
   }
 
-  # TODO: figure out why this gets installed anyway on new systems
   # xfce4-power-manager asks for admin authentication on login in order to
   # "change laptop display brightness"
   package { 'xfce4-power-manager':
```

Comment:
we can probably remove this `ensure => absent` now since it won't get installed in the first place?